### DISCUSSION TITLE
96E64AEE-EA92-4E00-6A3B-7D8D09E6B7D7

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#61] [General]
# 96E64AEE-EA92-4E00-6A3B-7D8D09E6B7D7

10199 Chariklo is the largest confirmed centaur known (measuring at 232 km in diameter). It orbits the Sun between Saturn and Uranus, grazing the orbit of Uranus. On 2014-Mar-26, astronomers announced the discovery of two rings around Chariklo by observing a stellar occultation, making it the first minor planet known to have rings. Due to its size, it is classified as a dwarf planet candidate (according to Mike Brown)

https://celestia.mobi/addon?item=96E64AEE-EA92-4E00-6A3B-7D8D09E6B7D7&lang=en

**Comment by Oskar0O0** [comment]
I have 2 chariklo's clipping through each other

**Reply by SevenSpheres** [reply]
This should be fixed now.
